notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>Are we using -q because there's a confirmation step we need to skip? Or to minimize verbose output? if confirmation step, ok, if minimizing output, let's remove it and let the user see </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>In a Vertex AI Workbench - Managed Notebook environment, interactive shell is not supported. <code>-q</code> is same as <code>--quiet</code> and it disables all interactive prompts when running gcloud commands. If input is required, defaults will be used, or an error will be raised. <a href="https://cloud.google.com/sdk/gcloud/reference/#--quiet" target="_blank">link</a></p><p> </p><p>Another way that I found was&nbsp;<code>printf "yes" | </code>&nbsp;approach. It will show all console outputs as normal, and submit the value&nbsp;<code>yes</code>&nbsp;when prompted.</p>